webapps/ui/admin/client/scripts/pages/diagnostics.js

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+var fs = require('fs');
+
+var template = fs.readFileSync(__dirname + '/diagnostics.html', 'utf8');
+
+var Controller = [
+  '$scope',
+  '$translate',
+  'camAPI',
+  'Notifications',
+  function($scope, camAPI) {
+    var telemetryResource = camAPI.resource('telemetry');
+
+    telemetryResource.fetchData({}, function(err, res) {
+      $scope.data = res;
+    });
+  }
+];
+
+module.exports = [
+  'ViewsProvider',
+  function PluginConfiguration(ViewsProvider) {
+    ViewsProvider.registerDefaultView('admin.system', {
+      id: 'diagnostics-system-settings',
+      label: 'DIAGNOSTICS',
+      template: template,
+      controller: Controller,
+      priority: 950,
+      access: [
+        'AuthorizationResource',
+        function(AuthorizationResource) {
+          return function(cb) {
+            AuthorizationResource.check({
+              permissionName: 'ALL',
```

Comment:
The telemetry data request requires READ (2) permission on the SYSTEM (21) resource.
I'm not sure how the page should behave if the user doesn't have the permission but one way would be to add it here.

Reply:
What do you think if we always display the page but show an error notification when the request fails due to missing permissions? I think this would be more robust and more aligned with our behavior in Cockpit.